src/transformers/configuration_auto.py

```diff
@@ -147,6 +150,7 @@
         ("pegasus", "Pegasus"),
         ("blenderbot", "Blenderbot"),
         ("marian", "Marian"),
+        ("barthez", "BARTHEZ"),
```

Comment:
```suggestion
        ("barthez", "BARThez"),
```
(This is used to display the proper name of the models in auto-generated docstrings.)